### PR TITLE
Bug: Fix flag name typo

### PR DESCRIPTION
**Pull request by SunburstEnzo**
### What

I noticed the flag in the Example project was using `OEPNSWIFTUI` instead of `OPENSWIFTUI`

### How

Changed the flag name in Build Settings and ensured it was changed everywhere else (only `ViewController.swift` from what I could tell)
Screenshot 2024-04-06 at 2 24 59 pm

Let me know if I missed anything!

### CODE REVIEW

File: Example/HostingExample/ViewController.swift [hunk 1]
```diff
@@ -6,7 +6,7 @@
 //
 
 import UIKit
-#if !OEPNSWIFTUI
+#if !OPENSWIFTUI
 import SwiftUI
 #else
 import OpenSwiftUI
```
Comment (Kyle-Ye) [inline Example/HostingExample/ViewController.swift]: Thanks for the PR.

Would you mind also updating the order here in this PR?

```swift
#if OPENSWIFTUI
import OpenSwiftUI
#else
import SwiftUI
#endif
```

Comment (SunburstEnzo) [reply Example/HostingExample/ViewController.swift]: Sure thing!

**Review by Kyle-Ye — APPROVED** [review]
LGTM. Thanks

**Comment by Kyle-Ye** [comment]
The repo requires all commits to be signed. You need to recommitting the commits and force push to solve it.

See more info here https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits

**Comment by Kyle-Ye** [comment]
Still showing as "Unverified". Maybe you need to upload your GPG key to your GitHub account.

image

**Comment by Kyle-Ye** [comment]
I've cherry-picked your changes and will merge this in #64. (Reserving you as the commit author)

Close this PR. Sorry for that.

**Comment by SunburstEnzo** [comment]
No problem at all and apologies for the delay – should now show as Verified hopefully
Screenshot 2024-04-06 at 3 57 17 pm

**Pull request closed without merging**